<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormaPrijemnica</class>
 <widget class="QMainWindow" name="FormaPrijemnica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>716</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Prijemnica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Primalac dobara</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prijemnica Br.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pošiljalac-davalac dobara</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="brojPrijemnice">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nova">
       <property name="text">
        <string>Nova</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="izmeni">
       <property name="text">
        <string>Izmeni</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="obrisi">
       <property name="text">
        <string>Obriši</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tabelaPosiljalac">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tabelaPrimalac">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tabelaPrijemnica">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
